### ISSUE TITLE
In browsing mode / virtual cursor – Missing effective way of navigating between cards 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14115]
This ticket is output of the user testing.

**Description**: Arrows don’t work for navigation between cards in browsing mode / virtual cursor and user doesn’t have any other way of effective navigation from card to card. It would help when the name of each card would be marked up as a heading or when cards would be marked up as list items. In such case user could navigate between cards by commands for headings or list items: H and I. 

**So far we discussed:**
Headings navigation would depends on the card usage. Some cards don’t need have headings inside.  

**Potential action:**
Recommendation/Best practice consumer can do headings inside the card.
